softgroup/services/profile/profile-impl/src/main/java/com/softgroup/profile/impl/handler/GetContactProfilesHandler.java

```diff
@@ -18,13 +27,45 @@
         extends AbstractRequestHandler<GetContactProfilesRequest,
             GetContactProfilesResponse> implements ProfileRequestHandler {
 
+    @Autowired
+    private ContactService contactService;
+
+    @Autowired
+    private Mapper mapper;
+
     public String getName() {
         return "get_contact_profiles";
     }
 
 
     @Override
     public Response<GetContactProfilesResponse> doHandle(Request<GetContactProfilesRequest> request) {
-        return null;
+        GetContactProfilesRequest requestData = request.getData();
+        GetContactProfilesResponse getContactProfilesResponse = new GetContactProfilesResponse();
+
+        String profileId = request.getRoutingData().getProfileId();
+        List<ContactDTO>  contactDTOS = getContactDTOs(profileId);
+        getContactProfilesResponse.setContactDTOS(contactDTOS);
```

Comment:
@VadymPolyanski What should return get_contact_profiles command? Did you read api?